pkgs/development/python-modules/yaramod/default.nix

```diff
@@ -0,0 +1,89 @@
+{ lib
+, buildPythonPackage
+, cmake
+, fetchFromGitHub
+, pytestCheckHook
+, libxcrypt
+, pythonOlder
+, gtest
+}:
+
+let
+  pog = fetchFromGitHub {
+    owner = "metthal";
+    repo = "pog";
+    rev = "b09bbf9cea573ee62aab7eccda896e37961d16cd";
+    hash = "sha256-El4WA92t2O/L4wUqH6Xj8w+ANtb6liRwafDhqn8jxjQ=";
+  };
+  pybind11 = fetchFromGitHub {
+    owner = "pybind";
+    repo = "pybind11";
+    rev = "refs/tags/v2.9.2";
+    hash = "sha256-O3bkexUBa+gfiJEM6KSR8y/iVqHqlCFmz/9EghxdIpw=";
+  };
+  json = fetchFromGitHub {
+    owner = "nlohmann";
+    repo = "json";
+    rev = "refs/tags/v3.9.1";
+    hash = "sha256-THordDPdH2qwk6lFTgeFmkl7iDuA/7YH71PTUe6vJCs=";
+  };
```

Comment:
Why are we not using nixpkgs versions of json and pybind11?

Reply:
Honestly, not sure about pybind11. I tested it earlier and it was broken with nixpkgs vesion of pybind11. Now I've changed it to the nixpkgs version, and it passes all my tests (and the included pytest tests). So probably I was confused/tested a wrong version.

For json I didn't know how to get the "single header include" for it, but found an example in another package from nixpkgs, so I replaced that.